app/lib/three_scale/timed_value.rb

```diff
@@ -12,10 +13,11 @@ def self.set(key, value, expire_in_secs = 60*10)
     redis.expire(key, expire_in_secs)
   end
 
-  protected
+  class << self
+    protected
 
-  def self.redis
-    Redis::Namespace.new(:timed_value, redis: System.redis)
```

Comment:
We don't store this kind of keys under the `timed_value` namespace anymore. We'll use the same DB number `System.redis` does. I don't think nobody uses `TimedValue` anyway. Tests pass, though.